payloadcms/payloadcms-dev/docker-compose.yml

```
@@ -0,0 +1,44 @@
+version: "3"
+
+services:
+  payload:
+    image: node:18-alpine
+    ports:
+      - "3000:${PORT:-3000}"
+    volumes:
+      - .:/home/node/app
+      - node_modules:/home/node/app/node_modules
+    working_dir: /home/node/app/
+    command: sh -c "yarn install && yarn dev"
+    depends_on:
+      - mongo
+    env_file:
+      - .env
+
+  mongo:
+    image: mongo:5.0
+    ports:
+      - "27017:${MONGO_PORT:-27017}"
+    command:
+      - --storageEngine=wiredTiger
+    volumes:
+      - data:/data/db
+    logging:
+      driver: none
+
+  localstack:
+    container_name: "${PROJECT:-localstack-main}"
+    image: localstack/localstack
```

Can we set a specific version so it doesn't download the latest one always? 

```suggestion
    image: localstack/localstack:3.0
```